features/webgl2.yml

```diff
@@ -241,25 +234,6 @@ compat_features:
   - api.WebGL2RenderingContext.viewport
   - api.WebGL2RenderingContext.waitSync
   - api.WebGLQuery
```

Comment:
Even though this is titled `WebGLXXX` it is explicitly WebGL2... Same for some of the other top level APIs